FlightStreamDeck.Logics/Actions/GenericToggleAction.cs

```diff
@@ -353,14 +353,31 @@ protected override Task OnKeyDown(ActionEventArgs<KeyPayload> args)
 
             if (holdEvent.HasValue)
             {
-                timer = new Timer { Interval = settings.HoldValueRepeat ? 400 : 1000 };
+                int frequency = GetRepeatFrequencyForMobiflightSettings(settings, 400);
+                timer = new Timer { Interval = settings.HoldValueRepeat ? frequency : 1000 };
+                
                 timer.Elapsed += Timer_Elapsed;
                 timer.Start();
             }
 
             return Task.CompletedTask;
         }
 
+        private int GetRepeatFrequencyForMobiflightSettings(GenericToggleSettings settings, int defaultFreq)
```

Comment:
May I know if there is any reason this is specific for MobiFlight? For me the beauty of MobiFlight is that it doesn't need specific support from the plugin: it just works seamlessly with SimConnect.